system/jlib/jstats.cpp

```diff
@@ -377,7 +377,14 @@ static StringBuffer & formatLoad(StringBuffer & out, unsigned __int64 value)
 static StringBuffer & formatSkew(StringBuffer & out, unsigned __int64 value)
 {
     //Skew stored as 10000 = perfect, display as percentage
-    return out.appendf("%.2f%%", ((double)(__int64)value) / 100.0);
+    const __int64 & sval = (__int64) value;
+    const double percent = ((double)sval) / 100.0;
+    if (sval >= 10000 || sval <= -10000)
```

Comment:
for the code reader, I think adding back comments that say // >= 100% or similar helps.